src/data-tutorials/users.tsx

```diff
@@ -957,6 +957,13 @@ const Users: User[] = [
     website: 'blog/tutorials/truffle-and-celo-the-ultimate-guide-to-deploy-celo-dapps-with-truffle',
     tags: ['beginner','foundation', 'truffle'],
   },
+  {
+    title: 'Exploring Celo in the Gaming Industry for In-Game Assets and P2P Trading',
+    description: 'Learn how Celo is utilized in the Gaming Industry for In-Game Assets and P2P Trading',
+    preview: require('./showcase/beginner/Exploring-Celo-in-the-Gaming-Industry-for-In-Game-Assets-and-P2P-Trading.png'),
```

Comment:
change it to small case

Reply:
I just edited this, i think this might have been the issue. Thanks again